### ISSUE TITLE
Make main menu usable even if tile images don't load

### ISSUE THREAD

**Issue by PGijsbers** [PGijsbers/pgijsbers.github.io#10]
If images don't load for whatever reason, then the titles (`ABOUT`,...) and alt-text should be easily readable.

**Comment by PGijsbers** [comment]
Clarification: the menu items do work ([disable images](chrome://settings/content/images?search=images)), but alt-text overlaps with button text, and the "buttons" are not easily identifiable as such: 
<img width="743" alt="image" src="https://github.com/PGijsbers/pgijsbers.github.io/assets/15890747/fd445f59-3cea-4af4-bad0-56d6434910f2">
